src/World.cpp

```diff
@@ -46,12 +46,13 @@ std::vector<Light*> World::pruned_lights(vec3 point){
 }
 bool World::can_see_light(vec3 point, Light light)
 {
-    vec3 newDirection = light.position - point;
+    vec3 newDirection = *light.position - point;
     Ray r = Ray(&point, &newDirection);
-    return equal( get_intersect( r ), background );
```

Comment:
can you change this back to `equal()` so that we're using GLM built-in vector comparison?